packages/headless/src/api/search/search/search-request.test.ts

```diff
@@ -170,4 +171,34 @@ describe('search request', () => {
     state.fields.fetchAllFields = true;
     expect(buildSearchRequest(state).request.fieldsToInclude).not.toBeDefined();
   });
+
+  it('when there are no cq expressions in state, cq is undefined', () => {
+    expect(buildSearchRequest(state).request.cq).toBe(undefined);
```

Comment:
I see a lot of tests like this one here that check for either the absence of a key or a key with the value `undefined`. Should we narrow down those tests to checking only the absence of a key?
E.g.:
```suggestion
    expect(buildSearchRequest(state).request).not.toHaveProperty('cq');
```

Reply:
Update: playing with this locally, I realized is we lose type safety with this approach. If we ever change the name of the property (e.g. search api v4 renames `cq` to `cachedFilter`), then this test would still pass, instead of typescript alerting us to change `cq` to the new name.

What do you think?